### DISCUSSION TITLE
Mutationpp python package update

### DISCUSSION THREAD

**Discussion by ggange** [mutationpp/Mutationpp#202] [Q&A]
Hello guys, as you saw in my pull request ( #188 ) I added the possibility to use the python package in thermal non-equilibrium (i.e. possibility to set more than one T as input to setState()) but the package to be downloaded from https://test.pypi.org/project/mutationpp/ does not seem to have been updated since July 2021 (and indeed the problem is still present there)... Am I missing something? In #179 @rdbisme was saying so:

"The package on PyPI is updated automatically via Github actions ([here](https://github.com/mutationpp/Mutationpp/blob/master/.github/workflows/wheels.yml)). Basically the package is rebuilt and uploaded automatically when the code is tagged. So there's no need to update the local package, it will be done automatically by the CI system.

So the only thing needed here is to merge your code update, and then tag a new version. Github will take care of the rest."

How do I tag the new code? 
Thank you in advance :D

**Comment by rdbisme** [comment]
Ehi @ggange, thanks for joining the discussion. The people allowed to tag the code are the maintainers of the project. I will work as soon as I have a bit of free time to tag a new version, and also to promote the python package to the official PyPI registry and not the test one.

**Comment by ggange** [comment]
Thank you! :)
